packages/testing/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@clerk/testing",
+  "version": "0.1.0",
```

Comment:
I'd like us to publish this directly in the 1.x version range, not 0.x. This just makes it easier to follow semver and I don't see a reason why we should keep this "experimental" so to speak